api/migrations/versions/0618a0517021_.py

```diff
@@ -0,0 +1,70 @@
+"""empty message
```

Comment:
again, should be one migration per PR. also when you generate migrations use -m flag with the short description, so this is not an empty message.